crates/cast/src/args.rs

```diff
@@ -694,11 +696,18 @@ pub async fn run_command(args: CastArgs) -> Result<()> {
             &mut std::io::stdout(),
         ),
         CastSubcommand::Logs(cmd) => cmd.run().await?,
-        CastSubcommand::DecodeTransaction { tx } => {
-            let tx = stdin::unwrap_line(tx)?;
-            let tx = SimpleCast::decode_raw_transaction(&tx)?;
```

Comment:
these two lines we should keep exactly as is

Reply:
done :)